#information of objects

object_info[] obj
float64 dt
point3d dX

